vue/src/modules/Workspace/article/ArticleView.vue

```diff
@@ -0,0 +1,202 @@
+                <b-media class="article-header align-items-center">
+                  <b-img v-if="currentArticle.image_url" class="article-image" :src="currentArticle.image_url"/>
+                  <i v-else class="fas fa-folder-open fa-4x"/>
+                </b-media>
+              </div>
+              <div class="col-md-8 col-sm-12">
+                <div>
+                  <h3 class="mt-0">{{ currentArticle.title }}</h3>
+                </div>
+                <div v-html="currentArticle.body"></div>
+              </div>
+            </div>
+          </b-card-body>
+        </b-collapse>
+      </b-card>
+      <div class="p-3">
+        <div class="row" v-if="lastFolder && isFolder && multipleFolders">
+          <div class="col-sm-6 col-lg-4">
+            <h4 class="border-bottom pb-1 mb-3">{{ $t('Articles') }}</h4>
+            <div class="article-list">
+              <content-spinner :show="loading" :text="$t('Loading...')" class="h-100"/>
+              <no-data :model="filteredArticles" :loading="loading" :text="$t('There are no articles')"></no-data>
+              <div class="article-list">
+                <ArticleChildItem
+                  v-for="(article, index) in filteredArticles" :index="index" :model="article" :key="article.id"/>
+              </div>
+            </div>
+          </div>
+          <div class="col-sm-6 col-lg-4">
+            <h4 class="border-bottom pb-1 mb-3">{{ $t('Folders') }}</h4>
+            <div class="article-list">
+              <content-spinner :show="loading" :text="$t('Loading...')" class="h-100"/>
+              <no-data :model="filteredFolders" :loading="loading" :text="$t('There are no folders')"></no-data>
+              <div class="article-list">
+                <ArticleChildItem
+                  v-for="(folder, index) in filteredFolders" :index="index" :model="folder" :key="folder.id"/>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="row" v-else-if="!isFolder">
+          <div class="col-sm-12 col-lg-6">
+            <b-card class="article-item mb-3" no-body>
+              <template v-slot:header>
+                <h5 class="mb-0">{{ currentArticle.title }}</h5>
+              </template>
+              <b-card-body>
+                <div class="article-content">
+                  <div v-if="currentArticle.body" class="article-body" v-html="currentArticle.body"></div>
+                  <b-img v-if="currentArticle.image_url" class="article-image" :src="currentArticle.image_url"/>
+                  <no-data v-else :model="currentArticle" :loading="loading"
+                           :text="$t('There is no description')">
+                  </no-data>
+                </div>
+              </b-card-body>
+            </b-card>
+            <div class="article-list">
+              <ArticleChildItem
+                v-for="(article, index) in filteredArticles" :index="index" :model="article" :key="article.id"/>
```

Comment:
@skavt  Do not use directly folder.id as key. It may not be unique in the page, use itself and some unique string name combination.